test/libsolidity/syntaxTests/constructor/interface_constructor_new.sol

```diff
@@ -1,7 +1,7 @@
 interface I {
-	constructor();
+	constructor() public;
```

Comment:
Should be external (I assume there is another test without any visibility specifier).

Reply:
Fixed. Yes, there's a test for that: ``constructor_no_visibility.sol``